### ISSUE TITLE
Repeating group Widget does not support rules written in .yml file

### ISSUE THREAD

**Issue by bennsimon** [OpenSRP/opensrp-client-native-form#300]
Repeating group only support inline rules

**Comment by rahimkhalid** [comment]
Was your issue fixed? because I am also facing the same issue.
Can you please help me out.

**Comment by bennsimon** [comment]
Yeah, now the repeating groups support rules engine for relevance however not for calculation and constraints.
 
An example `repeating groups support rules engine for relevance ` can be found in the native forms sample app on the repeating groups widget sample.

**Comment by rahimkhalid** [comment]
Ok, can you please give some example regarding inline calculation, because I'm not able to find any from the provided examples.

**Comment by bennsimon** [comment]
Inline calculation is not supported

**Comment by bennsimon** [comment]
@rahimkhalid you can try this following this example
`       
 "constraints": [
          {
            "type": "numbers_selector",
            "ex": "lessThanEqualTo(., step1:numbers_selector)"
          }
        ]
`

**Comment by rahimkhalid** [comment]
Thank's it worked
